v0/destinations/salesforce/transform.js

```diff
@@ -235,11 +235,33 @@ async function getSalesforceIdFromPayload(message, authorizationData) {
         leadObjectId = leadQueryResponse.data.searchRecords[0].Id;
       }
     }
+    if (leadObjectId) {
+      const convertedDetails = await axios.get(
```

Comment:
this should be wrapped in try/catch with appropriate error handling

Reply:
Updated.